src/units/drawer.hpp

```diff
@@ -58,6 +58,10 @@ class unit_drawer
 	int hex_size;
 	int hex_size_by_2;
 
+	bool is_selected_hex(const map_location& loc) const;
```

Comment:
This needs documentation explaining that it can return true for hexes other than the selected one; hopefully a better name will emerge from writing the documentation.

Reply:
It's a private function, but I agree - name bad...